lang/java/avro/src/test/java/org/apache/avro/io/TestBlockingIO2.java

```diff
@@ -99,6 +89,6 @@ public static Collection<Object[]> data() {
 
         { 100, 1, "{c1sK5e10}" }, { 100, 1, "{c1sK5U1S10}" }, { 100, 1, "{c1sK5f10S10}" }, { 100, 1, "{c1sK5NS10}" },
         { 100, 1, "{c1sK5BS10}" }, { 100, 1, "{c1sK5IS10}" }, { 100, 1, "{c1sK5LS10}" }, { 100, 1, "{c1sK5FS10}" },
-        { 100, 1, "{c1sK5DS10}" }, });
+        { 100, 1, "{c1sK5DS10}" }, }).map(Arguments::of);
```

Comment:
You're using a different strategy here than above. Both are good, but I usually prefer just one. Let's keep this however, given the number of lines that would change otherwise.